<ui version="4.0" >
 <class>TTVideoFileInfoWidget</class>
 <widget class="QWidget" name="TTVideoFileInfoWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>137</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>620</width>
    <height>80</height>
   </size>
  </property>
  <property name="baseSize" >
   <size>
    <width>620</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="spacing" >
    <number>4</number>
   </property>
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QPushButton" name="pbVideoOpen" >
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="videofileinfowidget.qrc" >:/pixmaps/pixmaps/fileopen_24.xpm</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbVideoDelete" >
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="videofileinfowidget.qrc" >:/pixmaps/pixmaps/cancel_18.xpm</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="videoListView" >
     <property name="focusPolicy" >
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="contextMenuPolicy" >
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>Filename</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Length</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Resolution</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Ratio</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Framerate</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Bitrate</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>vbvBuffer</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="videofileinfowidget.qrc" />
 </resources>
 <connections/>
</ui>
